horovod/common/wire/message_generated.h

```diff
@@ -1,19 +1,3 @@
-// Copyright 2019 Uber Technologies, Inc. All Rights Reserved.
```

Comment:
While this file is generated from the flatc, I think we should keep the existing copyright/license header intact.